examples/src/examples/misc/gizmos/example.mjs

```diff
@@ -226,6 +231,32 @@ function createColorMaterial(color) {
     return material;
 }
 
+// scene settings
+app.scene.ambientLight = new pc.Color(0.2, 0.2, 0.2);
+
+// create screen
+const screen = new pc.Entity('screen');
+screen.addComponent('screen', {
+    referenceResolution: new pc.Vec2(1280, 720),
+    scaleBlend: 0.5,
+    scaleMode: pc.SCALEMODE_BLEND,
+    screenSpace: true
+});
+app.root.addChild(screen);
+
+const instr = new pc.Entity('instructions');
+instr.setLocalPosition(0, 200, 0);
+instr.addComponent('element', {
+    pivot: new pc.Vec2(0.5, 0.5),
+    anchor: new pc.Vec4(0, 1, 1, 0.8),
+    margin: new pc.Vec4(0, 0, 0, 0),
+    fontAsset: assets.font.id,
+    fontSize: 18,
+    text: 'Translate (1), Rotate (2), Scale (3)\nWorld/Local (X)\nPerspective (P), Orthographic (O)',
+    type: pc.ELEMENTTYPE_TEXT
+});
+screen.addChild(instr);
```

Comment:
Is this nice and readable on mobile?

Reply:
The text seems very pixelated on mobile as do a lot of the other text examples

Reply:
I think you just need to do:

```
app.graphicsDevice.maxPixelRatio = window.devicePixelRatio;
```